kolena/detection/_internal/test_suite.py

```diff
@@ -211,28 +212,28 @@ def description(self, description: str) -> None:
             """
             Update the description of the test suite.
 
-            :param description: the new description of the test suite
+            :param description: The new description of the test suite.
             """
             self._description = description
 
         @validate_arguments(config=ValidatorConfig)
         def add(self, test_case: BaseTestCase) -> None:
             """
-            Add the provided :class:`kolena.detection.TestCase` to the test suite.
+            Add the provided [`TestCase`][kolena.detection.TestCase] to the test suite.
             If a different version of the test case already exists in this test suite, it is replaced.
 
-            :param test_case: the test case to add to the test suite
+            :param test_case: The test case to add to the test suite.
```

Comment:
Stray thought -- given that you're going through the effort of making param structure consistently full sentences (caplitalized, with periods), should we specify or link something as a style guide in the `CONTRIBUTING.md`?

Reply:
Good idea. Will add this to the backlog — want to make sure that the style guide is complete before committing anything.